tls/s2n_server_hello.c

```diff
@@ -89,7 +90,7 @@ static int s2n_server_add_downgrade_mechanism(struct s2n_connection *conn) {
     return 0;
 }
 
-int s2n_server_hello_recv(struct s2n_connection *conn)
+static int s2n_parse_server_hello(struct s2n_connection *conn)
```

Comment:
nit: would `s2n_server_hello_parse` be a better name, and in keeping with the verb being the last (`_recv`, `_send`, for example).

Reply:
Makes sense. I've updated the function name.